docs/source/getting_started.rst

```diff
@@ -72,22 +70,6 @@ install the latest version in your own custom conda environment.
 
 First, activate conda or install it if it's not available. Details vary amongst machines.
 
-Compy
------
-    ::
-
-     module load anaconda3/2019.03
-     source /share/apps/anaconda3/2019.03/etc/profile.d/conda.sh
-
-
-NERSC
------
-    ::
-
-     module load python/3.7-anaconda-2019.10
-     source /global/common/cori_cle7/software/python/3.7-anaconda-2019.10/etc/profile.d/conda.sh
```

Comment:
I think we can remove these `conda` instructions since the details vary so widely between machines and between users. For instance, my current command to get `conda` on Cori is `source /global/homes/f/forsyth/miniconda3/etc/profile.d/conda.sh`.

Reply:
I don't recommend the procedure that was listed here so I concur with getting rid of this.  There is no added benefit that I'm aware of in using someone else's anaconda module as a starting point rather than installing your own Miniconda or Mambaforge.